easybuild/easyconfigs/m/MACE/MACE-0.3.8-foss-2023a-CUDA-12.1.1-ASE-3.24.0.eb

```diff
@@ -18,7 +19,7 @@ dependencies = [
     ('PyTorch-bundle', '2.1.2', versionsuffix),
     ('e3nn', '0.4.4', versionsuffix),
```

Comment:
This needs to be changed too, to avoid:
```
Missing dependencies: PyTorch-bundle/2.1.2-foss-2023a-CUDA-12.1.1-ASE-3.24.0, e3nn/0.4.4-foss-2023a-CUDA-12.1.1-ASE-3.24.0 (no easyconfig file or existing module found
```